js/lib/beautify.js

```diff
@@ -170,6 +170,27 @@
         var beautifier = new Beautifier(js_source_text, options);
         return beautifier.beautify();
     }
+    
+    function verifyOperatorPosition(opPosition) {
+        var validPositionValues = [undefined];
+        Object.keys(OPERATOR_POSITION).forEach(function(pos) {
```

Comment:
Not sure if this is compatible with older browsers  (IE is a pain). 

Here's a polyfill to work around it:
http://tokenposts.blogspot.com.au/2012/04/javascript-objectkeys-browser.html

But, you might have to do this instead:
```js 
opPosition = opPosition || OPERATOR_POSITION. before_newline;
if (!OPERATOR_POSITION[opPosition]) {
    throw new Error();
}
return opPosition;
```